Dockerfile

```diff
@@ -1,4 +1,13 @@
-FROM        node:10-alpine
+FROM        quay.io/app-sre/centos:7
```

Comment:
Can we try to use centos8? It has way less vulnerabilities, as per https://quay.io/repository/app-sre/centos?tag=latest&tab=tags

Reply:
centos8 unable to find centos-release-scl-rh

will fix this later.